cluster.js

```diff
@@ -6,6 +6,7 @@ const log = require('./logger');
 
 const workers = [];
 let restart = true;
+let child;
```

Comment:
using let here as a use-before-defined error would be thrown if defined in scope below.